src/main/java/com/knu/buga1chuk/others/DirectoryCheck.java

```diff
@@ -35,10 +43,12 @@ public static void main(String[] args) {
                 String messageFormat = "Error: %s;";
                 String error = e.getMessage();
                 String message = String.format(messageFormat, error);
-                System.out.println(message);
+                //System.out.println(message);
+                LOG.info(message);
```

Comment:
Pay attention here. (it is log in catch)

Reply:
Fix